docs/Quick-Start-Guide/Architecture.md

```diff
@@ -107,6 +107,45 @@ mirror:
 # gluster volume create test-volume replica 3 transport tcp server1:/exp1 server2:/exp2 server3:/exp3 server4:/exp4 server5:/exp5 server6:/exp6
 volume create: test-volume: success: please start the volume to access data
 ```
+4. **Dispersed Glusterfs Volume** - Dispersed volumes are based on
+erasure codes. It stripes the encoded data of files, with some redundancy added,
+across multiple bricks in the volume. You can use dispersed volumes to
+have a configurable level of reliability with minimum space waste.
+The number of redundant bricks in the volume can be decided by clients while
+creating the volume. Redundant bricks determines how many bricks can be lost
+without interrupting the operation of the volume.
+
+Create a dispersed volume:
+
+```
+# gluster volume create test-volume [disperse [<COUNT>]] [disperse-data <COUNT>] [redundancy <COUNT>] [transport tcp | rdma | tcp,rdma] <NEW-BRICK>
+```
+
+**For example**, three node dispersed volume with level of redundancy 1, (2 + 1):
+
+```console
+# gluster volume create test-volume disperse 3 server1:/exp1 server2:/exp2 server3:/exp3
```

Comment:
Don't we need to add either disperse-data or redundancy here?

Reply:
Same as above comment

Reply:
Done